src/www/electron_index.html

```diff
@@ -21,11 +21,9 @@
     <title>Outline</title>
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=no" />
     <link rel="stylesheet" href="style.css" />
-    <script src="bower_components/outline-i18n/index.js"></script>
-    <script src="bower_components/punycode/punycode.min.js"></script>
-    <script src="bower_components/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
I think you still need the webcomponents polyfill.

Reply:
Thanks, added!